WordPress/gutenberg
# Adds a README.md file to the project scaffolded by create-block

## What?

Adds a `README.md` template file to the `create-block` package. This file contains an overview of the project scaffolded by `create-block` and descriptions of the files and directories contained within the project. There are also links to external information sources.

## Why?

Useful as a learning aid and as a reference when creating a new project with the `create-block` package.

## Testing Instructions
To test locally run each of the following commands from within `packages/create-block/lib`:

```
node index.js test-create-block
node index.js --variant dynamic test-create-block-dyn
```

Check that the `README.md` files generated each contain different content for static and dynamic blocks and that they accurately reflect the respective projects.

_Remember to delete the `test-create-block` and `test-create-block-dyn` directories when done!_

## Discussion point

This file will not be generated if the `--no-plugin` flag is defined. Should lines 59 to 84 instead be put in a separate `README.md` file in [the `block` directory](https://github.com/WordPress/gutenberg/tree/trunk/packages/create-block/lib/templates/block)?
